src/ui/SliderRange.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Range } from "rc-slider";
+
+import "rc-slider/assets/index.css";
+
+import { sliderStyling } from "./Slider";
+import { StyledSliderRange } from "./StyledSliderRange";
+
+// Please check https://github.com/react-component/slider#user-content-common-api
+// for all available props.
+const SliderRange = (props) => {
+  const [minMaxValues, setMinMaxValues] = React.useState({
+    min: props.defaultValue[0],
+    max: props.defaultValue[1],
```

Comment:
You should destructure props:

```js
const { defaultValue: [min, max] } = props;
```

Even better, should be destructured in the function args if possible.